base/missing.jl

```diff
@@ -109,7 +109,7 @@ max(::Any,     ::Missing) = missing
 # Rounding and related functions
 for f in (:(ceil), :(floor), :(round), :(trunc))
     @eval begin
-        ($f)(::Missing, digits::Integer=0, base::Integer=0) = missing
+        ($f)(::Missing; digits::Integer=0, base=0) = missing
```

Comment:
why remove `base::Integer` ?

Reply:
Just trying to sync with the docstring [here](https://github.com/JuliaLang/julia/blob/80516ca20297a67b996caa08c38786332379b6a5/base/floatfuncs.jl#L47-L120). I guess non-integer bases are, in principle, a [thing](https://en.wikipedia.org/wiki/Non-integer_representation), and the `Missing` implementation will always return `missing` anyway. Happy to put `::Integer` back in if that's preferred.

Reply:
AFAIK, the respective julia functions don't support non-integer bases, so it would be best to leave it so that a proper method error is thrown.

Reply:
I suspect the type was just omitted from the docstring for simplicity.